identity-verification/module_2/2.2 Onboarding IDV API Example.ipynb

```diff
@@ -0,0 +1,624 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Onboarding Use Case (1:1 Verification)\n",
+    "-----\n",
+    "\n",
+    "Here we are going to take a look at the Onboarding case. Here a user is asked to present an image of a trusted identification document, then snap a selfie image. The following diagram details the process. \n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "![Onboarding](onboarding_process.png \"Onboarding\")\n",
+    "\n",
+    "1. user presents an image of an identification document like a drivers license or passport\n",
+    "2. user snaps a selfie which will be used to compare to the drivers license\n",
+    "3. system detects a face in the identification document and performs quality checks \n",
+    "4. system detects a face in the selfie and performs quality checks\n",
+    "5. system compares the identification image to the selfie image \n",
+    "    - if the similarity is above the specified threshold then we say that the faces match \n",
+    "    - if the similarity is below the specified threshold then  we say that the faces DON'T match\n",
+    "6. system checks selfie against known onboarded users and fraudsters. \n",
+    "7. indexes the user and the user is onboarded. \n",
+    "\n",
+    "###### Sample Images from \"Labeled Faces in the Wild\":\n",
+    "-----\n",
+    "*Gary B. Huang, Manu Ramesh, Tamara Berg, and Erik Learned-Miller.\n",
+    "Labeled Faces in the Wild: A Database for Studying Face Recognition in Unconstrained Environments.\n",
+    "University of Massachusetts, Amherst, Technical Report 07-49, October, 2007* [pdf](\"http://vis-www.cs.umass.edu/lfw/lfw.pdf\")"
```

Comment:
```suggestion
    "University of Massachusetts, Amherst, Technical Report 07-49, October, 2007* [pdf](\"https://vis-www.cs.umass.edu/lfw/lfw.pdf\")"
```

Link to HTTPS instead of HTTP.

Reply:
done